app/src/main/java/com/sunasterisk/musixmatch/service/MusicService.java

```diff
@@ -27,6 +42,31 @@
     private int mStateLoop;
     private Context mContext;
 
+    private enum ACTIONS {
+        ACTION_PLAY,
```

Comment:
```suggestion
        PLAY,
```
tránh duplicate từ action